enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,360 @@
+
+### Goals
+
+1. Add a new field placementStrategy to the placement APIs to let end uer define the roll out strategy for the selected clusters.
+
+### Non-Goals
+
+1. The new Placement APIs field placementStrategy has no control over applying the workloads.
+1. The new Placement APIs field placementStrategy has no control over scheduling the workloads.
+
+## Proposal
+
+### User Stories
+
+#### Story 1: As end user I want to define a subset of the selected clusters as a affinity clusters.
+#### Story 2: As end user I want to divide the selected clusters to groups then a workloads applier APIs can use those cluster groups to roll out the workloads. 
+
+### Placement placementStrategy API
+
+PlacementStrategy's ClustersGroupPercentage field define a percentage of the total selected clusters that will divide the clusters to groups(batches). As example; for a total 100 clusters selected, clustersGroupPercentage equal to 20% will divide the selected clusters into 5 groups each group consist of 20 clusters. ClustersGroupPercentage default value is None. NumberOfClustersPerGroup is the max number of managedClusters can be exist in a single cluster group. ClustersGroupPercentage and NumberOfClustersPerGroup cannot be used at the same time. If ClustersGroupPercentage is defined it will override the NumberOfClustersPerGroup. AffinityClusterGroup is used to identify a subset of the managedClusters. The AffinityClusterGroup has its own label selector and affinityGroupPercentage.
+
+```go
+type PlacementSpec struct {
+	// ClusterSets represent the ManagedClusterSets from which the ManagedClusters are selected.
+	// +optional
+	ClusterSets []string `json:"clusterSets,omitempty"`
+
+	// Predicates represent a slice of predicates to select ManagedClusters. The predicates are ORed.
+	// +optional
+	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+            ...
+ 
+	// PlacementStrategy define affinity clusters as subset of the selected clusters and divide the selected clusters to groups.
+	// +optional
+	DecisionGenerator DecisionGenerator `json:"decisionGenerator,omitempty"`
```

Comment:
looks like by mistake I approved the suggest changes , I would prefer DecisionStrategy